### PR TITLE
Page for explaining sx126x sxc127x incompatibility

### PR DESCRIPTION
**Pull request by olliw42**
title says it

attempt at a page with that info
might be linked too at a larter point from respective locations

### CODE REVIEW

File: docs/SX126x_SX127x_INCOMPATIBILITY.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# mLRS Documentation: SX126x and SX127x Incompatibility #
+
+([back to main page](../README.md))
+
+mLRS hardware using the sx126x and sx127x LoRa chipsets are not compatible with each other, and cannot be operated together (they don't bind).
```
Comment (jlpoltrack) [inline docs/SX126x_SX127x_INCOMPATIBILITY.md:5]: Perhaps 'mLRS hardware using the SX126x and SX127x LoRa chipsets are **not** compatible with each other and will not bind.'

I prefer SX in caps to match the datasheet.

File: docs/SX126x_SX127x_INCOMPATIBILITY.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+This is for two reasons:
+- the SF6 spreading factor, which is used by mLRS for the 19 Hz mode, is incompatible for the sx126x and sx127x chipsets. 
+- the sx127x does not support the SF5 spreading factor which is used for the mLRS 31 Hz mode.
+
```
Comment (jlpoltrack) [inline docs/SX126x_SX127x_INCOMPATIBILITY.md:10]: Perhaps:

This is for two reasons:

1. The 31 Hz mode uses spreading factor 5 (SF5), which is only available on the SX126x chipset.
2. The 19 Hz mode uses spreading factor 6 (SF6), which was modified on the SX126x chipset making it incompatible with the SX127x chipset.

File: docs/SX126x_SX127x_INCOMPATIBILITY.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+This is the relevant part in the datasheet of the sx126x chipset:
```
Comment (jlpoltrack) [inline docs/SX126x_SX127x_INCOMPATIBILITY.md:11]: Same thought ...SX126x...

File: docs/SX126x_SX127x_INCOMPATIBILITY.md (hunk 1 above)
Comment (olliw42) [reply docs/SX126x_SX127x_INCOMPATIBILITY.md]: done

File: docs/SX126x_SX127x_INCOMPATIBILITY.md (hunk 2 above)
Comment (olliw42) [reply docs/SX126x_SX127x_INCOMPATIBILITY.md:10]: done

File: docs/SX126x_SX127x_INCOMPATIBILITY.md (hunk 3 above)
Comment (olliw42) [reply docs/SX126x_SX127x_INCOMPATIBILITY.md]: done

**Comment by olliw42** [comment]
excellent suggestions. Thx!

**Pull request merged**